sla/sla-core/sla-personalization/pom.xml

```diff
@@ -53,6 +59,11 @@
 			<groupId>com.sun.jersey</groupId>
 			<artifactId>jersey-client</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>it.polimi.modaclouds.qos-models</groupId>
+			<artifactId>qos-models</artifactId>
+		</dependency>
```

Comment:
As you know this library is deprecated, we should move to the **rules** module of Tower 4Clouds, version 0.2.3 ( https://github.com/deib-polimi/tower4clouds/releases/download/v0.2.3/rules-0.2.3-jar-with-dependencies.jar ). We already discussed this in pvt and maybe the best solution is to put this T4C dependency (along with the repository) in the root pom, since it is shared by the monitor modules and SLA. Anyway right now I'm placing this dependency in the modules' pom, where required, can you do the same here at least for now?